src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -866,6 +866,17 @@
             "default|ipv4_unicast|21.0.0.0/8": {
             }
         },
+        "BGP_NEIGHBOR": {
```

Comment:
Having two `BGP_NEIGHBOR` entries in this file will not work. You cannot have two of the same keys in most softwares, this will attract bugs. (I just tested this in redis and it does not work)

Reply:
will have 2 entries in BGP_NEIGHBOR table one with just IP and another one with VRF & IP to ensure both the tables parsed without any issues.

Reply:
I understand that we can't have two entries (one with just IP and another one with VRF+IP as keys) together, will check with @praveen-li  on how to accommodate two different configs in the sample_config_db.json if required.

Reply:
We do not need 2 BGP_NEIGHBOR tables in config?, As per yang models, it is same container, which allows both type of entries.